pages/home/home.html

```diff
@@ -1,19 +1,22 @@
+<!DOCTYPE html>
 <section id="main-content">
   <md-card layout-fill style="position: relative;">
     <div id="home-page-welcome-image-container" flex layout="column" layout-fill class="oppia-foundation-clearfix">
-      <img ng-src="https://placekitten.com/3000/1200" class="home-page-welcome-image">
-      <md-card-content layout="column" layout-align="center center" class="home-page-welcome-image-overlay horizontal-padding">
-        <div class="home-page-welcome-text-overlay">
-          <h1 hide-xs hide-sm class="oppia-foundation-horizontal-padding">
-            We're building a global classroom
-          </h1>
+      <img ng-src="/static/images/home_cheering_child.png" class="home-page-welcome-image">
```

Comment:
I don't think you need ng-src here, since no part of the image is being interpolated. src will do.

Reply:
Oh wow, good catch there. Thanks!
I probably did this in other html file and need to fix those errors over there too.
Will patch them separately in a different PR!